### DISCUSSION TITLE
Tags - feature request

### DISCUSSION THREAD

**Discussion by ririko5834** [anonaddy/anonaddy#458] [Feature Requests]
It would be nice if there would be an option to make custom tags in admin settings (that you can set up name and colour). And then similarly as receipts 
![image](https://github.com/anonaddy/anonaddy/assets/64729293/a0416ae4-04ce-463b-a09f-52a2cd5d699d)
tags should be displayed in the table as a badge with the set colour. There should also be an option to see all aliases that have the specific tag and search them by tags.

This is good to have it more organised especially if you have more email aliases for different stuff and want to have it organised by color.

**Comment by willbrowningme** [comment]
Thanks, this is already on the roadmap - https://github.com/anonaddy/anonaddy/projects/1#card-52584414
